tests/unit/etl/LoadBalancerTests.cpp

```diff
@@ -107,30 +107,32 @@ TEST_F(LoadBalancerConstructorTests, forwardingTimeoutPassedToSourceFactory)
     makeLoadBalancer();
 }
 
-TEST_F(LoadBalancerConstructorTests, fetchETLState_Source0Fails)
+TEST_F(LoadBalancerConstructorTests, fetchETLState_SourceAllFail)
```

Comment:
Probably better to name tests a bit more natural `fetchETLState_AllSourcesFail`.